tests/functional/common.sh

```diff
@@ -109,9 +109,11 @@ function configure_oioswift() {
 function run_script() {
   if "$1"; then
     printf "${GREEN}\n${1}: OK\n${NO_COLOR} ($2)"
+    return 0
   else
     RET=1
```

Comment:
RET is not used

Reply:
RET is a global variable and used for the final exit